internal/pkg/cmd/flag_resolver.go

```diff
@@ -31,11 +31,6 @@ func (r *FlagResolverImpl) ValueFrom(source string, prompt string, secure bool)
 		if prompt == "" {
 			return "", ErrNoValueSpecified
 		}
-		if yes, err := r.Prompt.IsPipe(); err != nil {
```

Comment:
This seemed like an unecessary restriction. It forces users to type "-" if they want to pipe input. As far as I know, this isn't a standard in most CLIs, so we should remove this as not to confuse users.